pkgs/development/gnuradio-modules/bladeRF/default.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  mkDerivation,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  boost,
+  doxygen,
+  gmp,
+  gnuradio,
+  libbladeRF,
+  mpir,
+  osmosdr,
+  python,
+  spdlog,
+}:
+
+mkDerivation {
+  name = "gr-bladeRF";
+  version = "unstable-2023-11-20";
```

```suggestion
  pname = "gr-bladeRF";
  version = "0-unstable-2023-11-20";
```
name is always generated from pname and version.